docs_user/modules/proc_migrating-mon-from-controller-nodes.adoc

```diff
@@ -45,15 +44,15 @@ additional networks.
         template: templates/single_nic_vlans/single_nic_vlans_storage.j2
 ----
 
-. Run the provisioning command:
+.. Provision the `CephStorage` nodes:
 +
 ----
 $ openstack overcloud node provision \
   -o overcloud-baremetal-deployed-0.yaml --stack overcloud-0 \
   --network-config -y --concurrency 2 /home/stack/metalsmith-0.yam
```

Comment:
Should `/home/stack/metalsmith-0.yam` be changed to `/home/stack/metalsmith-0.yaml` ? That is, .yam > .yaml.